x-pack/filebeat/Jenkinsfile.yml

```diff
@@ -35,9 +35,17 @@ stages:
             branches: true     ## for all the branches
             tags: true         ## for all the tags
         stage: extended
-    build:
-        mage: "mage build test"
-        withModule: true       ## run the ITs only if the changeset affects a specific module.
+    unitTest:
+        mage: "mage build unitTest"
+        withModule: true
```

Comment:
I think we don't need this particular flag, this is only required for the ITs to identify what module has been changed and run only for that particular module if possible.

Reply:
You are right, I must have copied an entire block wrong. Will send it ASAP